25.Blur/main.cpp

```diff
@@ -4,182 +4,500 @@
 
 #define _NBL_STATIC_LIB_
 #include <nabla.h>
-#include "../common/QToQuitEventReceiver.h"
-#include "nbl/ext/ScreenShot/ScreenShot.h"
 
-#include "nbl/ext/Blur/CBlurPerformer.h"
+#include "../common/CommonAPI.h"
+#include "nbl/ext/ScreenShot/ScreenShot.h"
+// #include "nbl/ext/CentralLimitBoxBlur/CBlurPerformer.h"
 
 using namespace nbl;
-using namespace core;
-
-
-//!Same As Last Example
-class MyEventReceiver : public QToQuitEventReceiver
-{
-		using Base = QToQuitEventReceiver;
-		const float BLUR_RADIUS_MIN = 0.f, BLUR_RADIUS_MAX = 0.5f;
+using namespace nbl::core;
+using namespace nbl::asset;
+using namespace nbl::video;
 
-	public:
-		ext::Blur::CBlurPerformer* blurPerf = nullptr;
-
-		bool OnEvent(const SEvent& event)
-		{
-			if (!Base::OnEvent(event))
-			if (event.EventType == nbl::EET_MOUSE_INPUT_EVENT && blurPerf)
-			{
-				float r = blurPerf->getRadius() + event.MouseInput.Wheel/500.f;
-				blurPerf->setRadius(std::max(BLUR_RADIUS_MIN, std::min(r, BLUR_RADIUS_MAX)));
-				//return true;
-			}
-
-			return false;
-		}
-};
+#if 0
```

Comment:
actually remove dead code, if I want to see old code I always have `git diff`

Reply:
Resolved.